nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractRecordProcessor.java

```diff
@@ -99,6 +108,8 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
 
         final RecordReaderFactory readerFactory = context.getProperty(RECORD_READER).asControllerService(RecordReaderFactory.class);
         final RecordSetWriterFactory writerFactory = context.getProperty(RECORD_WRITER).asControllerService(RecordSetWriterFactory.class);
+        final boolean dropEmptyFiles = context.getProperty(DROP_EMPTY_FILES).isSet()?
```

Comment:
Checkstyle violation here, the question mark should be on the next line (or the whole expression on one line)